api.md

```diff
@@ -2874,45 +2865,48 @@ view_definition                |  SELECT foo.a,
 
 ```
 ---
-## timescaledb_information.continuous_aggregate_stats [](timescaledb_information-continuous_aggregate_stats)
+## timescaledb_information.compression_settings [](timescaledb_information-compression_settings)
 
-Get information about background jobs and statistics related to continuous aggregates.
+Get information about compression related settings for hypertables.
+The view lists the orderby and segmentby columns used by compression.
 
-#### Available Columns [](timescaledb_information-continuous_aggregate_stats-available-columns)
+#### Available Columns [](timescaledb_information-compression_settings-available-columns)
 
 |Name|Description|
 |---|---|
-|`view_name`| User supplied name for continuous aggregate. |
-|`completed_threshold`| Completed threshold for the last materialization job.|
-|`invalidation_threshold`| Invalidation threshold set by the latest materialization job|
-|`last_run_started_at`| Start time of the last job|
-|`last_run_status` | Whether the last run succeeded or failed |
-|`job_status`| Status of the materialization job . Valid values are ‘Running’ and ‘Scheduled’|
-|`last_run_duration`| Time taken by the last materialization job|
-|`next_scheduled_run` | Start time of the next materialization job |
-| `total_runs` | The total number of runs of this job |
-| `total_successes` | The total number of times this job succeeded |
-| `total_failures` | The total number of times this job failed |
-| `total_crashes` | The total number of times this job crashed |
-
-#### Sample Usage [](timescaledb_information-continuous_aggregate_stats-examples)
-
-```sql
-SELECT * FROM timescaledb_information.continuous_aggregate_stats;
--[ RECORD 1 ]----------+------------------------------
-view_name              | contagg_view
-completed_threshold    | 1
-invalidation_threshold | 1
-job_id                 | 1003
-last_run_started_at    | 2019-07-03 15:00:26.016018-04
-last_run_status        | Success
-job_status             | scheduled
-last_run_duration      | 00:00:00.039163
-next_scheduled_run     | 2019-07-03 15:00:56.055181-04
-total_runs             | 3
-total_successes        | 3
-total_failures         | 0
-total_crashes          | 0
+| `table_schema` | Schema name of the hypertable |
+| `table_name` | Table name of the hypertable |
+| `attname` | Name of the column used in the compression settings |
+| `segmentby_column_index` | Position of attname in the compress_segmentby list |
+| `orderby_column_index` | Position of attname in the compress_orderby list |
+| `orderby_asc` | (BOOLEAN) True if this is used for order by ASC, False for order by DESC |
+| `orderby_nullsfirst` | (BOOLEAN) True if nulls are ordered first for this column, False if nulls are ordered last|
+
+
+#### Sample Usage [](timescaledb_information-compression_settings-examples)
+
+```sql
+CREATE TABLE hypertab (a_col integer, b_col integer, c_col integer, d_col integer);
```

Comment:
Maybe include a column that _isn't_ in order/compress by, so user realizes that not all columns are listed in the below view?  (i.e., they might otherwise have false impression)

```suggestion
CREATE TABLE hypertab (a_col integer, b_col integer, c_col integer, d_col integer, e_col integer);
```